BoxContentSDK/BoxContentSDK/BOXContentSDKConstants.h

```diff
@@ -364,6 +372,12 @@ extern NSString *const BOXAPIEventStreamTypeChanges;
 extern NSString *const BOXAPIEventStreamTypeSync;
 extern NSString *const BOXAPIEventStreamTypeAdminLogs;
 
+// API Recent Items Constants
+extern NSString *const BOXAPIRecentItemsTypeShared;
```

Comment:
Having seen 'listType' it seems that this should actually be 'BOXAPIRecentItemsListTypeShared' ?